.azure/hpu-tests-pl.yml

```diff
@@ -73,19 +73,19 @@ jobs:
       displayName: 'Check the driver status'
 
     - bash: |
-        python -m pytest -sv test_accelerator.py --forked --junitxml=hpu1_test-results.xml
+        python -m pytest -sv tests_pytorch/test_accelerator.py --forked --junitxml=hpu1_test-results.xml
```

Comment:
this should be parametrized or we dont want to test also fabric this way?

Reply:
fabric tests are part of hpu-tests-fabric